lua/telescope/pickers.lua

```diff
@@ -975,6 +1049,7 @@ function Picker:close_existing_pickers()
   end
 end
```

Comment:
I cant comment for `close_existing_pickers` but it does what the function tells you. If someone what start telescope while telescope is still open it would close the already existing instance. Like if a user wants to go from one picker into another with an action.

(We should probably do `actions._close(prompt_bufnr, true)` tho, with pcall obv, because this function is currently used to fix an bug we have, also see `actions.run_builtin`